app/src/main/res/values/styles.xml

```diff
@@ -21,4 +21,8 @@
         <item name="android:navigationBarColor">@android:color/black</item>
     </style>
 
+    <style name="ProgressBarTheme" parent="ThemeOverlay.AppCompat.Light">
+        <item name="colorAccent">@color/home_tx_value_negative_bg</item>
```

Comment:
I guess this just sets the progress bar color - not sure if it's better to define it as a theme, there's an example in the code to do it programmatically, see UiUtil.kt.